src/frontend/package.json

```diff
@@ -83,7 +83,7 @@
     "@types/react-datepicker": "4.1.7",
     "buffer": "6.0.3",
     "clipboard": "2.0.8",
-    "converse.js": "<8",
+    "converse.js": "~8.0.1",
```

Comment:
```suggestion
    "converse.js": "8.0.1",
```

We prefer having pin version like other dependencies in this file.

Reply:
I'm not changing converse's version in this PR anymore. However, i will do as you say for the second one.